_posts/2023-08-15-OLS-joining-forces-with-communities-to-broaden-participation-in-open-science.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: OLS joining forces with communities to broaden participation in open science
+authors:
+- nyasita
+- landimi2
+- lauracion
+- npalopoli
+- karvovskaya
+- malvikasharan
+- estherplomp
+- pazbc
+- anelda
+image: /images/2023-16-08-SouthAfrica-pic-OLS-8-blog-post.jpg
+photos:
+  name: Photo by Anelda Van der Walt
+---
+
+
+_OLS is happy to announce our collaboration with multiple international organisations and communities to deliver the Open Seeds OLS-8 programme_ 
+-----
+
+**Authors:** Laurah Nyasita Ondari, Laura Acion, Nicolás Palopoli, Malvika Sharan, Andela Van de Valt, Lena Karvovskaya, Laura Ascenzi, Esther Plomp, Mike Landi Kofia and Paz Bernaldo.
```

Comment:
Do we need that list if we have it in the top?

Reply:
oh only added that because for example Laura Ascenzi isn't on github

Reply:
but wait, she might be.... I'll double check

Reply:
@NPDebs could you
-  add Laura to the people as if she is mot on GitHub
- add the corresponding id to the authors list in the metadata of the post
- remove the extra "authors" line

Thanks a lot

Reply:
Sure thing! 👍

Reply:
I've added [Laura](https://openlifesci.org/people#aliaslaurel). She was staring at us the whole time. 😄

Reply:
Thanks a lot